backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java

```diff
@@ -68,12 +66,10 @@
 import org.apache.logging.log4j.Logger;
 import org.jdbi.v3.core.statement.Query;
 
-public class FutureProjectDAO {
+public class FutureProjectDAO extends UACApisUtil {
```

Comment:
Could we use composition instead of inheritance? Seems like a more logical approach as we're not implementing abstract methods. It sounds like the DAO _has_ some utils to deal with UAC, vs the DAO _is_ some utils to deal with UAC.

Reply:
Updated PR with composition instead of inheritance. I hope it will works now. :)